### DISCUSSION TITLE
Anti-Kick Malfunction

### DISCUSSION THREAD

**Discussion by Rankism** [winstxnhdw/lc-hax#413] [Q&A] [answered]
I've been using Anti-Kick for a little while but I can't get it to work. I turn on Anti-Kick via the `\` key but I'm still able to be kicked in games. Don't know if this information helps but I've also been joining games as Nameless, unknown, or something else (usually Nameless). Not sure if anyone else is having issues.

I know there's caveats but in #205, that issue's been fixed.

**Comment by winstxnhdw** [comment, marked as answer]
Hey, #205 was a false-alarm. It’s mentioned in later issues or discussion that it is not possible to be unkickable until you are the last player. That’s why anti-kick is combined with auto-join to help you achieve that.

**Reply by Rankism** [reply]
Hey, thanks for the quick reply. I'll keep that in mind next time I use it.

**Comment by winstxnhdw** [comment]
Read #220 as well.
